pkg/util/compressor/zstd/zstd.go

```diff
@@ -0,0 +1,72 @@
+package zstd
+
+import (
+	"io"
+	"sync"
+
+	"github.com/klauspost/compress/zstd"
+	"google.golang.org/grpc/encoding"
+)
+
+const Name = "zstd"
+
+func init() {
+	c := &compressor{}
```

Comment:
1. did we need to pre isntall zstd?
2. what compression level are we at ? I believe zstd has compression level 1-19.
For RPC, 3 might be perfect